share/harvesters/oai.py

```diff
@@ -103,3 +103,12 @@ def fetch_by_id(self, provider_id):
         url.args['metadataPrefix'] = self.metadata_prefix
         url.args['identifier'] = provider_id
         return etree.tostring(self.fetch_page(url)[0][0], encoding=str)
+
+    def metadata_formats(self):
```

Comment:
Consider moving to another PR since this isn't used my MODS (per our discussion).